src/test_typing_extensions.py

```diff
@@ -3262,6 +3262,9 @@ def __call__(self, *args: Unpack[Ts]) -> T: ...
         self.assertEqual(MemoizedFunc.__parameters__, (Ts, T, T2))
         self.assertTrue(MemoizedFunc._is_protocol)
 
+        # 3.11+ moves where this exception is thrown
+        # 3.11: TypeVarTuple.__typing_prepare_subst__
+        # 3.12+: typing._typevartuple_prepare_subst
         things = "arguments" if sys.version_info >= (3, 11) else "parameters"
```

Comment:
This is no longer necessary if we use `arguments` for all versions.
The `assertRaisesRegex` below should than also be updated to use `arguments` instead.
https://github.com/python/typing_extensions/blob/4fdc09ddb54be26580f68e26443a422c6024364c/src/test_typing_extensions.py#L3272-L3273